products/oracle-database.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Oracle
+category: db
+iconSlug: oracle
+permalink: /oracle-database
+versionCommand: show version
```

Comment:
I do not know in which context this command is valid. Maybe `SELECT BANNER_FULL FROM V$VERSION;` (taken from https://database.guide/7-ways-to-check-your-oracle-version/) would be better ?